src/components/FilterButtons.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Button from './ui/Button';
+
+function FilterButtons() {
+  return (
+    <>
+      <Button className="bottom_nav__right__btn">All</Button>
+      <Button className="bottom_nav__right__btn">Incomplete</Button>
+      <Button className="bottom_nav__right__btn">Complete</Button>
```

Comment:
Could we use a map here for not repeating the same things again ?
```suggestion
      const filterStates = [{label: 'All'}........]
      {filterStates.map((filterState) =>
          <Button className="bottom_nav__right__btn">{filterState.label}</Button>
        )}
```

Reply:
Done with map. Thanks